docs/core-api/PIN.md

```diff
@@ -25,6 +25,10 @@
   - [Options](#options-4)
   - [Returns](#returns-4)
   - [Example](#example-4)
+- TODO `ipfs.pin.remote.add(ipfsPath, [options])`
+- TODO `ipfs.pin.remote.ls([options])`
+- TODO `ipfs.pin.remote.rm(requestid)`
+- TODO `ipfs.pin.remote.~TBD~replace(requestid, ipfsPath, [options])`
```

Comment:
Q: does TS make docs automation useful, or do we need to write docs for below ones by hand?

```suggestion
- TODO `ipfs.pin.remote.add`
- TODO `ipfs.pin.remote.ls`
- TODO `ipfs.pin.remote.rm`
- TODO `ipfs.pin.remote.service.add`
- TODO `ipfs.pin.remote.service.ls`
- TODO `ipfs.pin.remote.service.rm`
```

Reply:
By hand for now - @hugomrdias is working on a types->docs generation tool but it's not ready yet.